Gruntfile.js

```diff
@@ -89,7 +89,8 @@ module.exports = function (grunt) {
 			"nsConfig = window." + config + " = window." + config + " || {};\n" +
 			"nsConfig.rootNamespace = '" + rootNamespace + "';\n" +
 			"nsConfig.fileName = '" + fileName + "';\n" +
-			"ns.version = '" + version + "';\n",
+			"ns.version = '" + version + "';\n" + 
+			"console.log('TAU version: " + version + "');",
```

Comment:
I think that better will be:
`ns.log("TAU version: " + ns.version + " profile: " + ns.info.profile);`

in `wrapEnd`